src/components/Dashboard/StatusPanel.js

```diff
@@ -15,15 +18,31 @@ class StatusPanel extends Component {
     this.state = {
       services: []
     };
+
+    this.onStatusChange = this.onStatusChange.bind(this);
   }
 
   async componentDidMount() {
     this.fetchStatuses();
+    initializeWebSocket(this.onStatusChange);
+  }
+
+  /**
+   * Implementation of real time status change
+   *
+   * @param service
+   */
+  onStatusChange(service) {
+    let services = this.state.services;
+    let index = _.findIndex(services, ['name', service.name]);
+
+    _.merge(services[index], service);
```

Comment:
This will mutate the `services` object. Need to do something like this http://ramdajs.com/docs/#update here:

```js
let updatedServices = update(index, service, serivices);

this.setState({ services: updatedServices })'